part-1/mailinglist/client/client.go

```diff
@@ -0,0 +1,78 @@
+package main
+
+import (
+	"context"
+	"log"
+	"mailinglist/proto"
+	"time"
+
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials/insecure"
+)
+
+func logResponce(res *proto.SubscriberResponse, err error) {
+	if err != nil {
+		log.Fatalf("error:%v", err)
+	}
+
+	if res.Subscriber == nil {
+		log.Printf("email not found")
+
+	} else {
+		log.Printf("responce:%v", res.Subscriber)
+	}
```

Comment:
обычно принято не создавать лишнее ветвление чтобы код было проще читать
```suggestion
	if res.Subscriber == nil {
		log.Printf("email not found")
                return
	} 
	log.Printf("responce:%v", res.Subscriber)
```